enumerable.rb

```diff
@@ -25,7 +24,7 @@ def my_select
     my_each do |item|
       result.push(item) if yield(item)
     end
-    puts result
+    result
   end
 
   def my_all?(obj = nil)
```

Comment:
- **_`#my_all?` when no block or argument is given returns true when none of the collection members are false or nil_**

- **_`#my_all?` when a class is passed as an argument returns false if any of the collection is not a member of such class_**

- **_`#my_all?` when a Regex is passed as an argument returns false if any of the collection does not match the Regex_**